src/hooks/auth/types.ts

```diff
@@ -9,3 +9,14 @@ export interface authHookReturn {
   login: () => Promise<void>
   logout: () => Promise<void>
 }
+
+
+export interface TurnkeyAuthHookReturn {
+  auth: Auth | undefined
+  active: boolean
+  activating: boolean
+  authAddr: string | undefined
+  name: string | undefined
+  login: (hasExistingPasskey: boolean) => Promise<void>
+  logout: () => Promise<void>
+}
```

Comment:
For better reusability, it should extend the existing base `authHookReturn` interface with the **login** part overridden

```suggestion
export interface TurnkeyAuthHookReturn extends Omit<authHookReturn, 'login'> {
  login: (hasExistingPasskey: boolean) => Promise<void>
}
```

Reply:
Minor (not related to your PR): `authHookReturn` => `AuthHookReturn`

Reply:
Fixed! Interesting feature